docs/event_details/screen_opened.md

```diff
@@ -0,0 +1,23 @@
+# screen_opened
+
+## When the event is called
+
+If you’d like to track when screen is opened.
```

Comment:
**suggestion**: It would be nice to add here also information, that event should be tracked when user navigates back to this screen from previous screen.
Also that this event is tracked, when user moves app into the background and bring it back.

**chore**: please ensure that those events are tracked in those scenarios.